config/locales/client.en.yml

```diff
@@ -413,6 +413,17 @@ en:
       invited_by: "Invited By"
       trust_level: "Trust Level"
       notifications: "Notifications"
+      desktop_notifications:
+        label: "Desktop Notifications"
+        not_supported: "Notifications are not supported on this browser. Sorry."
+        perm_default: "Turn On Notifications"
+        perm_denied_btn: "Permission Denied"
+        perm_denied_expl: "You have denied permission for notifications. Use your browser to enable notifications, then click the button when done. (Desktop: The leftmost icon in the address bar. Mobile: 'Site Info'.)"
+        disable: "Disable Notifications"
+        currently_enabled: "(Currently Enabled)"
+        enable: "Enable Notifications"
+        currently_disabled: "(Currently Disabled)"
```

Comment:
minor, but could you make this lower case? (currently disabled) -- title case should be used for titles.